editor/editor_system.cpp

```diff
@@ -318,8 +318,11 @@ void EditorSystem::drawDefaultUI(float deltaMilliseconds)
 				ImGui::Separator();
 				if (ImGui::BeginMenu("Create Scene"))
 				{
+					SceneID inputBaseID = 0;
 					ImGui::InputText("Scene Name", &newSceneName, ImGuiInputTextFlags_AlwaysInsertMode);
-					if (!newSceneName.empty() && ImGui::Button("Create") && !Scene::isReservedName(newSceneName))
+					ImGui::InputScalar("Scene BaseID", ImGuiDataType_U32, &inputBaseID);
+					Scene::SetBaseID(inputBaseID);
+					if (!newSceneName.empty() && ImGui::Button("Create"))
```

Comment:
Rewrite this line to:
https://github.com/sdslabs/Rootex/blob/a5a7735a8f7b31a1a1bfff9bee27e99433e483af/editor/editor_system.cpp#L322